server.sublime-workspace

```diff
@@ -0,0 +1,329 @@
+		"/home/emeka/Desktop/php/laravel/resources/views/welcome.blade.php",
+		"/home/emeka/Desktop/php/laravel/app/Http/routes.php",
+		"/home/emeka/Desktop/php/laravel/resources/views/emeka.blade.php~",
+		"/home/emeka/Project/javascript/number/index.html",
+		"/home/emeka/.config/sublime-text-3/Packages/User/Preferences.sublime-settings",
+		"/home/emeka/Project/javascript/number/function/f1.js",
+		"/home/emeka/Project/javascript/number/css.css",
+		"/home/emeka/.homestead/Homestead.yaml",
+		"/home/emeka/Desktop/Andela/num/index.html",
+		"/home/emeka/Desktop/Andela/num/function/f1.js",
+		"/home/emeka/Desktop/vag/Vagrantfile"
+	],
+	"find":
+	{
+		"height": 33.0
+	},
+	"find_in_files":
+	{
+		"height": 102.0,
+		"where_history":
+		[
+		]
+	},
+	"find_state":
+	{
+		"case_sensitive": false,
+		"find_history":
+		[
+			"assets",
+			"\n</",
+			"//'",
+			"display_errors"
+		],
+		"highlight": true,
+		"in_selection": false,
+		"preserve_case": false,
+		"regex": false,
+		"replace_history":
+		[
+		],
+		"reverse": false,
+		"show_context": true,
+		"use_buffer2": true,
+		"whole_word": false,
+		"wrap": true
+	},
+	"groups":
+	[
+		{
+			"sheets":
+			[
+			]
+		}
+	],
+	"incremental_find":
+	{
+		"height": 33.0
+	},
+	"input":
+	{
+		"height": 36.0
+	},
+	"layout":
+	{
+		"cells":
+		[
+			[
+				0,
+				0,
+				1,
+				1
+			]
+		],
+		"cols":
+		[
+			0.0,
+			1.0
+		],
+		"rows":
+		[
+			0.0,
+			1.0
+		]
+	},
+	"menu_visible": true,
+	"output.find_results":
+	{
+		"height": 0.0
+	},
+	"pinned_build_system": "",
+	"project": "server.sublime-project",
+	"replace":
+	{
+		"height": 62.0
+	},
+	"save_all_on_build": true,
+	"select_file":
+	{
+		"height": 0.0,
+		"last_filter": "",
+		"selected_items":
+		[
+			[
+				"post_tab",
+				"database/migrations/2015_09_08_222438_create_post_table.php"
+			],
+			[
+				"feeds",
+				"app/Http/Controllers/FeedsController.php"
+			],
+			[
+				"feed",
+				"app/Http/Controllers/feedsController.php"
+			],
+			[
+				"index/bo",
+				"resources/views/includes/index/home.blade.php"
+			],
+			[
+				"index.bl",
+				"resources/views/index.blade.php"
+			],
+			[
+				"rou",
+				"app/Http/routes.php"
+			],
+			[
+				"rout",
+				"vendor/symfony/routing/Route.php"
+			],
+			[
+				"home",
+				"resources/views/includes/index/home.blade.php"
+			],
+			[
+				"con",
+				"app/Http/Controllers/Controller.php"
+			],
+			[
+				".ev",
+				".env"
+			]
+		],
+		"width": 0.0
+	},
+	"select_project":
+	{
+		"height": 438.0,
+		"last_filter": "",
+		"selected_items":
+		[
+		],
+		"width": 333.0
+	},
+	"select_symbol":
+	{
+		"height": 0.0,
+		"last_filter": "",
+		"selected_items":
+		[
+		],
+		"width": 0.0
+	},
+	"selected_group": 0,
+	"settings":
+	{
+	},
+	"show_minimap": false,
+	"show_open_files": false,
+	"show_tabs": true,
+	"side_bar_visible": true,
+	"side_bar_width": 268.0,
+	"status_bar_visible": true,
+	"template_settings":
+	{
+	}
```

Comment:
Why did you commit this file?..You want to impose your workspace config on every one? :angry: